installer/pyinstaller/build-linux.sh

```diff
@@ -47,15 +48,32 @@ echo "Installing PyInstaller"
 
 echo "Building Binary"
 cd src
+if ! [ "$nightly_build" = "" ]; then
```

Comment:
Can we move nightly build check into a variable and use it in all places here?

Reply:
done.